List/index.d.ts

```diff
@@ -29,7 +29,11 @@ declare class ItemIcon extends MaterialComponent<IItemIconProps, {}> {
   getProxyClassName(props: IItemIconProps): string;
 }
 
-declare class ItemAvatar extends MaterialComponent<JSX.HTMLAttributes, {}> {}
+declare interface IItemAvatarProps extends JSX.HTMLAttributes {
+  'start-detail'?: boolean;
+  'end-detail'?: boolean;
+}
+declare class ItemAvatar extends MaterialComponent<IItemAvatarProps, {}> {}
```

Comment:
`ItemAvatar` extends from `ItemIcon`. Let's just extend it directly. There are also no props from `ItemAvatar` other than the inherited props.

If in the future we want more props on `ItemAvatar` we can do as `LinkItem`.